src/python/pants/backend/nfpm/native_libs/elfdeps/analyze_test.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import zipfile
+from pathlib import PosixPath
+from typing import cast
+
+import pytest
+import requests
+from _pytest.tmpdir import TempPathFactory
+from elfdeps import ELFAnalyzeSettings, ELFInfo, SOInfo
+from pytest import fixture
+
+from .analyze import ELFInfoAnalysis, analyze_directory, analyze_wheel, analyze_wheels_repo
```

Comment:
I think house style is full imports from the package root.

But then, if no linters complain then I won't either.